src/utils.ts

```diff
@@ -199,7 +208,14 @@ export function getBestLimitAskExcludePubKey(
 		oraclePriceData
 	);
 	for (const ask of asks) {
-		if (ask.userAccount?.equals(excludedUserAccount)) {
+		if (
+			ask.userAccount?.toBase58() === excludedPubKey ||
+			excludedUserAccountsAndOrder?.some(
```

Comment:
might be nice to put this on a separate if block so it's more readable

Reply:
done